src/commands/networks/createNetwork.ts

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { window } from 'vscode';
+import { IActionContext } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+import { wrapDockerodeENOENT } from '../../utils/wrapError';
+
+export async function createNetwork(_context: IActionContext): Promise<void> {
+
+    const name = await ext.ui.showInputBox({
+        value: '',
+        prompt: 'Name of the network'
+    });
+
+    const driverSelection = await ext.ui.showQuickPick(
+        [
+            { label: 'bridge' },
+            { label: 'host' },
+            { label: 'overlay' },
+            { label: 'macvlan' }
+        ],
+        {
+            canPickMany: false,
+            placeHolder: 'Select the network driver to use (default is "bridge").'
+        }
+    );
+
+    const result = <{ id: string }>await wrapDockerodeENOENT(() => ext.dockerode.createNetwork({ Name: name, Driver: driverSelection.label }));
+
+    window.showInformationMessage(`Network Created with ID ${result.id}`);
```

Comment:
Is ID really long and should potentially be truncated?

Reply:
Yeah the ID is quite long. Docker truncates the Network ID to 12 characters when you run `docker network ls`, so I'll truncate it to 12 characters too.